activities/Clock.activity/icons/second-hand.svg

```diff
@@ -0,0 +1,21 @@
+<svg id="SvgjsSvg1001" width="288" height="288" xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs"><defs id="SvgjsDefs1002"></defs><g id="SvgjsG1008" transform="matrix(0.9167,0,0,0.9167,11.99519256754698,11.995237874771647)"><svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 311.661 311.661" viewBox="0 0 311.661 311.661" width="288" height="288"><path d="M155.661,65.431c-50,0-90.8,40.8-90.8,90.8s40.8,90.4,90.8,90.4s90.4-40.8,90.8-90.8
```

Comment:
I'm not sure the gears is pertinent here.
I suggest to use two icons for this new button: one with three hands when the seconds hand is here, one with two hands when the seconds in not here.
Ensure that the icon is free/libre license or is derivative work on a free/libre icon.

Reply:
wouldn't it be better if we just add one button and then the message i.e the content it shows on hover is different if the seconds hand is hidden and vice versa? 
just a suggestion
will go ahead with what you think is correct

Reply:
May be I was not clear: yes the idea is to have one button with two different possible background icon (one with two hands, one with three hands). Regarding "on hover" event it don't works on touch devices so we can't really rely on it.

Reply:
yes I agree, will do the changes soon